engine/language-server/src/main/scala/org/enso/languageserver/protocol/json/JsonRpcProtocolFactory.scala

```diff
@@ -5,12 +5,21 @@ import org.enso.jsonrpc.{Protocol, ProtocolFactory}
 /** Factory creating JSON-RPC protocol. */
 final class JsonRpcProtocolFactory extends ProtocolFactory {
 
+  private[this] var _protocol: Protocol = _
```

Comment:
`private` should also work here, right? Just curious

Reply:
It will, yes. A bit of a habit of mine to make variables object private by default.